ValidationLibrary/Rules/HasNewestPtcsJenkinsLibRule.cs

```diff
@@ -43,6 +44,9 @@ public async Task Init(IGitHubClient ghClient)
             var versionFetcher = new ReleaseVersionFetcher(ghClient, "protacon", LibraryName);
             _expectedVersion = await versionFetcher.GetLatest();
             _logger.LogInformation("Rule {ruleClass} / {ruleName}, Newest version: {expectedVersion}", nameof(HasNewestPtcsJenkinsLibRule), RuleName, _expectedVersion);
+
+            var releases = await ghClient.Repository.Release.GetAll("protacon", LibraryName);
+            _latestReleaseUrl = releases[0].HtmlUrl;
```

Comment:
Functionality is fine, but perhaps a better way would be to extend `ReleaseVersionFetcher` as it already fetches the latest version, just return the `TagName` and the `HtmlUrl` (or even the DTO itself).